lib/realtime_web/controllers/broadcast_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule RealtimeWeb.BroadcastController do
+  use RealtimeWeb, :controller
+  alias RealtimeWeb.Endpoint
+  alias Realtime.GenCounter
+  alias Realtime.Tenants
+  action_fallback(RealtimeWeb.FallbackController)
+
+  defmodule Payload do
```

Comment:
You may have modeled this from me including a schema in the LiveView for the form...

But I think this definitely warrants going into it's own module in `realtime` (not `realtime_web`).

The use of changesets here I think is v cool btw. An easy way to respond with a batch of errors.

Reply:
kept it here because I wasn't sure where to move it to 😅 

will move it to `realtime` and yes it simplified a lot the implementation of error handling